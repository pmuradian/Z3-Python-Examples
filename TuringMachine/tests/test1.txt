5 2 5
a b c d e
i f
i
f
a i b i 0
b i c i 0
c i d i 0
d i e i 0
e i a f 1
6
a a a a a a