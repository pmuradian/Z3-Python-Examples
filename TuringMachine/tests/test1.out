YESa|i a| a| a| a| a| b|i a| a| a| a| a| c|i a| a| a| a| a| d|i a| a| a| a| a| e|i a| a| a| a| a| a| a|f a| a| a| a| 